sktime/utils/validation/series.py

```diff
@@ -61,11 +66,11 @@ def check_series(Z, enforce_univariate=False, allow_empty=False, allow_numpy=Tru
         _check_is_univariate(Z)
 
     # check time index
-    check_time_index(Z.index, allow_empty=allow_empty)
+    check_time_index(Z.index, allow_empty=allow_empty, index_type=index_type)
     return Z
 
 
-def check_time_index(index, allow_empty=False):
+def check_time_index(index, allow_empty=False, index_type=None):
```

Comment:
I like the idea of adding this here, I'd rename `index_type` to `enforce_index_type`

Reply:
done